src/rpc/net.cpp

```diff
@@ -193,7 +193,7 @@ UniValue getpeerinfo(const JSONRPCRequest& request)
 UniValue addnode(const JSONRPCRequest& request)
 {
     std::string strCommand;
-    if (request.params.size() == 2)
+    if (!request.params[1].isNull())
```

Comment:
> Add {} where appropriate.

Added as followup in https://github.com/bitcoin/bitcoin/pull/11050#issue-250186439. This PR is already doing a bunch of different things and has been reviewed by 4 people. I don't want to add noise, delay, and complexity sticking braces everywhere as well.